card/lib/card/format.rb

```diff
@@ -392,9 +392,12 @@ def prepare_nest opts
 
       opts ||= {}
       case
-      when opts.has_key?( :comment )                            ; opts[:comment]   # as in commented code
-      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''     # already out of view
-      when opts[:inc_name]=='_main' && !Env.ajax? && @depth==0  ; expand_main opts
+      when opts.has_key?( :comment )                                      ; opts[:comment]
+        # commented nest
+      when @mode == :closed && @char_count > Card.config.max_char_count   ; ''
+        # already out of view
+      when opts[:inc_name]=='_main' && show_layout? && @depth==0          ; expand_main opts
+        # why do we have these restrictions on main?
```

Comment:
In other words, to have two {{_main|...}} in a layout.  That could be handled if we have a spec for what we wanted.  Maybe two are ok, maybe one doesn't get the wrapper.